### PR TITLE
Add a chapter about Running or debugging gradle under Eclipse

### PR DESCRIPTION
**Pull request by zosrothko**
Any of the checked boxes below indicate that I took action:

- [x] Reviewed the [Contribution Guidelines](https://github.com/gradle/gradle/blob/master/.github/CONTRIBUTING.md).
- [x] Signed the [Gradle CLA](http://gradle.org/contributor-license-agreement/).
- [x] Ensured that basic checks pass: `./gradlew quickCheck`

For all non-trivial changes that modify the behavior or public API:

- [ ] Before submitting a pull request, I started a discussion on the [Gradle developer list](https://groups.google.com/forum/#!forum/gradle-dev),
      the [forum](https://discuss.gradle.org/) or can reference a [JIRA issue](https://issues.gradle.org/secure/Dashboard.jspa).
- [ ] I considered writing a design document. A design document can be
brief but explains the use case or problem you are trying to solve,
touches on the planned implementation approach as well as the test cases
that verify the behavior. Optimally, design documents should be submitted
as a separate pull request. [Samples](https://github.com/gradle/gradle/tree/master/design-docs)
can be found in the Gradle GitHub repository. Please let us know if you need help with
creating the design document. We are happy to help!
- [ ] The pull request contains an appropriate level of unit and integration
test coverage to verify the behavior. Before submitting the pull request
I ran a build on your local machine via the command
`./gradlew quickCheck <impacted-subproject>:check`.
- [ ] The pull request updates the Gradle documentation like user guide,
DSL reference and Javadocs where applicable.

Add a chapter about Running or debugging gradle under Eclipse. Fix typo for invoking gradlew.bat under Windows.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -17,9 +17,23 @@ git config user.email user@example.com
...
 ./gradlew idea       # *nix-based systems
-./gradlew.bat idea   # Windows
+gradlew idea         # Windows
 ```
-then "Open" it with IntelliJ. Unfortunately, Eclipse project import isn't straightforward so it's not recommended.
+then "Open" it with IntelliJ.
+
+You can generate the Eclipse projects by running
+```
+./gradlew eclipse   # *nix-based systems
```
Comment (eriwen) [inline .github/CONTRIBUTING.md]: Have you found this to work in practice? My understanding was that this doesn't currently do the right thing for the gradle/gradle build.

**Comment by eriwen** [comment]
@zosrothko Thank you. The content looks good except that I want to confirm with you that `./gradlew eclipse` works. Once you confirm that works on gradle/gradle I'll merge this.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (zosrothko) [inline .github/CONTRIBUTING.md]: Yes you are right. Running/Debugging the gradle build itself under Eclipse does not work, but this works for all others builds.

**Comment by zosrothko** [comment]
Yes, both
```
./gradlew eclipse     // on *nix
gradlew eclipse       // on windows
```
are working

**Comment by eriwen** [comment]
Merged with https://github.com/gradle/gradle/commit/02097f7f952c0e96ef1476bfd0fccb4b48a18205 — thank you @zosrothko!

**Pull request closed without merging**